Assets/Scripts/Interaction Scripts/MapInteractions.cs

```diff
@@ -1,420 +1,143 @@
-        {
-            IDLE, DRAGGING, MOVING, ROTATING, SCALING
-        }
-        // The radius of the table (assuming the table is circular)
-        public float tableRadius;
-        // The radius of the map (assuming the map is circular)
-        public float mapRadius;
-        // The circular table
-        public GameObject rotatingTable;
-
-        // Rotation stuff
-        public LinkedList<float> angles;
-        public bool handleHeldTrigger = false;
-        public static MapState mapState;
-        public OVRInput.Controller currentController;
-        private Vector3 oldVec;
-        private float movementAngle;
-        public float movementAngleDecay = .95f;
-
-        // Pointer Controller
-        // private GameObject controller;
-        // private VRTK.VRTK_StraightPointerRenderer pointer;
-
-        // Peru: 6/9/2020 : MapBox now completely in MapInteractions
-        public bool droneOutOfBounds = false;
-
-        // Mapbox Interactions
-        public GameObject citySim;
-        public GameObject rsf_roof;
-
-        public AbstractMap abstractMap;
-        private float initZoom_citySim = 21.0f;
-        private double initLat_citySim = 37.91532757;
-        private double initLong_citySim = -122.33805556;
-
-        public bool citySimActive = false;
-        private GameObject citySimParent;
-        private Vector3 initPosition_citySim;
-
-        public double currLat_citySim;
-        public double currLong_citySim;
-        private Vector3 currPosition_citySim;
-
-        private float minZoom_citySim = 0.0f;
-        private float maxZoom_citySim = 22.0f;
+        public ControllerState controllerState; /// Initiate a singleton of the ControllerState class. Assign it to the Controller GameObject.
+        public GameObject World; /// The World GameObject. All its children will be scaled, rotate and move with it.
+
+
+        private GameObject pivot; /// pivot is the center of the table
+
+        private Vector3 originalScale; ///originalScale is the original localScale of the world
+        // FIXME: Remove.
+        public Vector3 actualScale; ///actualScale is the relative localScale of the world in comparison to its original localScale
+        // FIXME: Remove.
+        private Vector3 currentScale; ///currentScale is the current localScale
+
+        public float MinimumScale; /// Let the user specify TODO
+        public float MaximumScale; /// Let the user specify TODO
+        private Vector3 minScale; ///This is the TODO of the originalScale of the world
+        private Vector3 maxScale; ///This is the TODO times the originalScale of the world
+
+        public float Speed; ///This is the speed at which the map can be moved at
+        public float RotationalSpeed; /// Rotation speed (in rev/s)
 
 
         // Use this for initialization
         void Start()
         {
-            //Terrain assignment
-            terrain = GameObject.FindGameObjectsWithTag("Ground");
```

Comment:
Do we no longer need this?

Reply:
No. It used to denote a ground plane. Was not reference anywhere. Maybe ISAACS v1 devs thought it could be useful in the future.